app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayInactiveTabsOnboardingBinding.kt

```diff
@@ -63,26 +65,29 @@ class TabsTrayInactiveTabsOnboardingBinding(
             settings.canShowCfr
 
     private fun createInactiveCFR() {
+        // Don't create a new dialog if one is already displayed
+        if (this::inactiveDialog.isInitialized)
+            return
```

Comment:
```suggestion
        if (this::inactiveDialog.isInitialized) return

```
Let's add a new line after this